Dockerfile

```diff
@@ -61,16 +69,14 @@ USER node
 
 ENTRYPOINT ["/usr/bin/tini", "--"]
 
-CMD ./node_modules/.bin/lc39 ./index.js --port=${HTTP_PORT} --log-level=${LOG_LEVEL} --prefix=${SERVICE_PREFIX} --expose-metrics ${EXPOSE_METRICS} --enable-tracing=${ENABLE_TRACING}
+CMD ["./node_modules/.bin/lc39", "./index.js", "--port", "${HTTP_PORT}", "--log-level", "${LOG_LEVEL}", "--prefix=${SERVICE_PREFIX}", "--expose-metrics", "${EXPOSE_METRICS}", "--enable-tracing=${ENABLE_TRACING}"]
```

Comment:
When changing this, the service does not start anymore logging the following error:

```
/home/node/app/node_modules/@mia-platform/lc39/lib/launch-fastify.js:76
    throw new Error('Prefix value is not valid')
          ^

Error: Prefix value is not valid
    at validatePrefix (/home/node/app/node_modules/@mia-platform/lc39/lib/launch-fastify.js:76:11)
    at launchFastify (/home/node/app/node_modules/@mia-platform/lc39/lib/launch-fastify.js:111:3)
    at launchFastifyWithFile (/home/node/app/node_modules/@mia-platform/lc39/lib/launch-fastify.js:97:10)
    at launchFastifyWithModuleOrFile (/home/node/app/node_modules/@mia-platform/lc39/lib/launch-fastify.js:161:12)
    at launchFromCli (/home/node/app/node_modules/@mia-platform/lc39/lib/launch-fastify.js:167:25)
    at Object.<anonymous> (/home/node/app/node_modules/@mia-platform/lc39/bin/cli.js:57:33)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)
    at Module.load (node:internal/modules/cjs/loader:1288:32)
    at Module._load (node:internal/modules/cjs/loader:1104:12)
```

Would it be possible to check what is causing this issue?

Reply:
From what I understood, as it is explained [here](https://www.docker.com/blog/docker-best-practices-choosing-between-run-cmd-and-entrypoint/), `exec` form of the `CMD` instruction does not perform variable _expansion_ (since the shell is not involved).
Consequently, parameters cannot be customized launching the container with specific environment variables

Reply:
I'm here with another weird trick, I've tested locally that `CMD ["./node_modules/.bin/lc39", "./index.js", "--port=${HTTP_PORT} --log-level=${LOG_LEVEL} --prefix=${SERVICE_PREFIX} --expose-metrics=${EXPOSE_METRICS} --enable-tracing=${ENABLE_TRACING}"]` this syntax works as intended. I don't know what weird interaction there is between tini and node and lc39 that if we pass every argument as a separate string it will not work but if they are grouped inside the same string the interpolation is working as intended 🤷🏻

Reply:
No, I was wrong, the trick wasn't working, I've reverted back to the exec syntax. We can ignore the warning that the buildx build process will output.

Reply:
Ok, thank you for the contribution and the clarification 👍🏻